pyqtgraph/parametertree/parameterTypes.py

```diff
@@ -833,7 +833,16 @@ def popupFilePicker(parent=None, winTitle='', nameFilter='', directory=None, sel
     else:
         fList = []
     if relativeTo is not None:
-        fList = [os.path.relpath(file, relativeTo) for file in fList]
+        # Forward slashes are returned by qt, so preserve this convention regardless
+        # of the os. Since os.path will always prefer os-flavored separators, use pathlib to consistently
+        # return posix flavor
+        newFileList = []
+        for file in fList:
+            # pathlib and darwin have case sensitivity issues using relative_to, so use os.path for this
+            relative = os.path.relpath(file, relativeTo)
+            relativePosix = Path(relative).as_posix()
```

Comment:
for this, I think we can use `os.fsdecode(filename)`

Reply:
Can probably do the whole thing in a list comprehension

```python
newFileList = [os.fsdecode(os.path.relpath(file, relativeTo)) for file in fList]
```

Reply:
does `fsdecode` turn backslash into forward slash? Don't see it in the docs

Reply:
@pijyoi @ixjlyons Looking for some sort of consensus on this. Either unix-style paths can be preserved (at the expense of converting from `str` -> `Path` -> `str` (`Path(relative).as_posix()`), or the return value changes to os-flavored paths if `relativeTo` is supplied. This is because `os.path.normpath` converts to os-flavored. But, `pathlib` wouldn't need to be used.

I'm in favor of risking type conversion to provide consistent path behavior, but that's mainly because I haven't experienced any negative effects from `pathlib` /  `Path` usage yet. @j9ac9k mentioned some inconsistencies with how `pathlib` treats casing, but I think that might be for operations that *create* paths -- in contrast, this usage would just be directly representing a path. For instance, on Windows calling `str(Path(filename))` still preserves casing. I could be wrong, though. Not sure what the best tests would be.